seed/tests/test_properties_serializers.py

```diff
@@ -331,10 +331,13 @@ def test_get_history(self):
         """Test get_history"""
         obj = mock.MagicMock()
         obj.state = self.property_state
+
+        data = self.serializer.get_history(obj)
```

Comment:
Hi @fablet -- This PR adds quantities to various fields in the seed database (basically adding units to certain fields). I hacked these tests to get them passing and I realized that this may cause issues. The results of the get_history is a serializer object but since it hasn't been passed to JSON yet, the encoding still includes Quantity fields. You can see my hack by forcing the quantity field to a magnitude. Anyway, I was wondering if you think this will cause issues? If so, any ideas on how to address this more elegantly?

Thanks!